src/Commands.js

```diff
@@ -37,6 +38,7 @@
             fired = [],
             prefix = "cmdId_",
             indexer = 0,
+            rethrow, //default true
```

Comment:
i understand and mostly agree with it but I think that simply swallowing errors isnt a good way to go about it especially with the only alternative being to put a function called "log" on the global object. 
I should at least be able to pass my own log function into the defaults so it will be used instead of the global one...